spinn_machine/machine.py

```diff
@@ -645,3 +645,15 @@ def total_cores(self):
         """
         return len([
             processor for chip in self.chips for processor in chip.processors])
+
+    def has_wrap_arounds(self):
+        """ If the machine has wrap around links
+
+        :return: True if wrap around links exist, false otherwise
+        :rtype: bool
+        """
+        if (self.max_chip_x == 2 and self.max_chip_y == 2) or \
+                (self.max_chip_x % 12 == 0 and self.max_chip_y % 12 == 0):
+            return True
+        else:
```

Comment:
Sorry to keep pushing new things... you don't need an else here, as the previous statement contains a return.

It isn't critical again, but it does read a bit better (I think).